Packs/Base/ReleaseNotes/1_15_9.md

```diff
@@ -0,0 +1,7 @@
+#### Scripts
+##### SanePdfReports
+- Update the Docker image to: *demisto/sane-pdf-reports:1.0.0.24891*
+- 6.5 includes the following updates:
+    - Fix overflow of text content at markdown sections
+    - Section Date update of date format
+    - Bug fixes of https://github.com/demisto/etc/issues/42120
```

Comment:
lets remove his bullet  (Bug fixes of https://github.com/demisto/etc/issues/42120)

because  this link is redirecting to a private repo

Reply:
done